js/controller.js

```diff
@@ -115,7 +115,12 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       "forceControlBarVisible": false,
       "timer": null,
       "errorCode": null,
-      "isSubscribed": false,
+
+      "isSubscribed": {
```

Comment:
@aldigun Ok. I've moved it to two separate flags, leaving the "isSubscribed" flag as it is on master.